src/foam/u2/PropertyBorder.js

```diff
@@ -160,7 +165,10 @@
           start().
             add(prop.view$.map(v => {
               // Add the Property's View
-              return this.E().add(prop.toE({ ...self.viewArgs, mode$: modeSlot}, this.__context__))
+              return this.E().add(prop.toE({
+                ...self.viewArgs,
+                mode$: modeSlot
+              }, this.__subContext__ ))
```

Comment:
Does just using 'this' work instead of this.__subContext__?

Reply:
Nope, element toE_ expects a context